### DISCUSSION TITLE
Theme - add default Background/Text colour

### DISCUSSION THREAD

**Discussion by Aderinom** [mantinedev/mantine#1317] [Feature requests]
Hello Mantine team!

Currently the MantineTheme colors only represent different colors ("blue","green","dark","grey", ...).
Components are bound to use a certain shade of some default color as base. 

e.x.
> Text supports special dimmed value in color prop. It sets color to theme.colors.dark[2] in dark theme and to theme.colors.gray[6] in light:  

This makes actually theming the page more difficult, since I have to either set the background and text color manually for every component. 

Would it be possible to either have default color props like (background, foreground, text...) which will be used by the components, or a parameter like the "primaryColor" param where you can set the default colorshade used by text  and background (f.e. "textColor")?

**Comment by rtivital** [comment]
You can customize Text and other components styles using Styles API on MantineProvider, it supports styles params from which you can determine what color was used for the Text – https://mantine.dev/theming/mantine-provider/#styles-on-mantineprovider

**Reply by Aderinom** [reply]
Yep, that works. Thank you!
